server/src/resolvers/mutation.js

```diff
@@ -1,237 +1,57 @@
-const APP_SECRET = process.env.SECRET || 'default';
-const {
-  cannotCreateUserMessage,
-  userNotFoundMessage,
-} = require('../error-messages');
-const expirationTime = '100h';
-const jwt = require('jsonwebtoken');
-const sha256 = require('sha256');
-const sendmail = require('sendmail')();
-
-const EMAIL_FROM = 'no-reply@tinysocial.SangGeonZZang.com';
-const EMAIL_TITLE = 'Action Required: Verify your email for the TinySocial';
-
-// Verify password string
-function passworldVerify(pw) {
-  let ret = true;
-  if (pw.length < 8) {
-    ret = false;
-  }
-  if (pw.match(/\d/g) === null) {
-    ret = false;
-  }
-  if (pw.match(/[a-zA-Z]/g) === null) {
-    ret = false;
-  }
-
-  return ret;
-}
-
 module.exports.Mutation = {
-  logout: async () => {
-  },
-
   signInWithGoogle: async (_, {googleId}, {dataSources}) => {
-    const user = await dataSources.mainAPI.findUser({googleId});
+    const user = await dataSources.mainAPI.findOneUser({googleId});
     if (user === null) {
       return {
         success: false,
-        message: userNotFoundMessage,
+        message: 'User not found. You have to sign up first.',
```

Comment:
Why are we going back to before? We've done the work to refactor out all custom strings in the previous state.